website/versioned_docs/version-0.10.0/gitops-run/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
+    "label": "GitOps Run",
+    "position": 5
+}
+  
```

Comment:
Is it required to have a "new line at the end of file"? I see the GitHub's message but I'm not sure if we take it into account.

Reply:
Was it autogenerated actually? I hadn't noticed I was in the versioned docs territory already. 🤣

Reply:
>On the other hand, this is only really relevant in pub quizzes - real world tools don't care.

Yes, totally!